sdk/storage/Azure.Storage.Files.Shares/src/Generated/FileRestClient.cs

```diff
@@ -9080,7 +9080,7 @@ public partial class ShareStatistics
         /// <summary>
         /// The approximate size of the data stored in bytes, rounded up to the nearest gigabyte. Note that this value may not include all recently created or recently resized files.
         /// </summary>
-        public int ShareUsageBytes { get; internal set; }
+        public long ShareUsageInBytes { get; internal set; }
```

Comment:
One example of this situation from BCL is Array.Length vs Array.LongLength, I'm not sure that we should use the same naming pattern but just FYI.